### DISCUSSION TITLE
Cannot read properties of undefined (reading 'on')

### DISCUSSION THREAD

**Discussion by djyt** [prb28/vscode-amiga-assembly#290] [General]
Hello,

Have been enjoying coding with this great extension. However, VSCode just auto updated itself, and now whenever I try to compile and run I receive the cryptic error dialog: 

Cannot read properties of undefined (reading 'on')

Does anyone have any tips as to how to avoid this?

![image](https://github.com/prb28/vscode-amiga-assembly/assets/2414449/3ca0820c-026c-437d-9f79-29ad53181c79)

**Comment by prb28** [comment]
I have the same error, let's create an issue !

**Comment by djyt** [comment]
Is it possible to move to a bug if it's not just me? FWIW, the project I'm working on is here - so very reliant on this:
https://eab.abime.net/showthread.php?t=116656

**Reply by prb28** [reply]
Great projet ! 
Yep the bug is opened (https://github.com/prb28/vscode-amiga-assembly/issues/291), I'm actually trying to find the cause.
